README.md

```diff
@@ -3,6 +3,55 @@
 This project contains a Java-Builder for Exasol error messages.
 The invocations of the Builder can be parsed by the [error-code-crawler-maven-plugin](https://github.com/exasol/error-code-crawler-maven-plugin).
 
+## Usage
+
+### Simple Messages
+```java
+ExaError.messageBuilder("E-TEST-1").message("Something went wrong.").toString();
+```
+result: `E-TEST-1: Something went wrong.`
+
+### Parameters
+
+```java
+ExaError.messageBuilder("E-TEST-2")
+  .message("Unknown input {{input}}.")
+  .parameter("input", 'unknown', "The illegal user input.").toString();
+```
+result: `E-TEST-2: Unknown input 'unknown'.`
+
+The optional third parameter for `parameter(placeholder, value, description)` is used by the [error-code-crawler-maven-plugin](https://github.com/exasol/error-code-crawler-maven-plugin) to generate a parameter description.
+
+The builder automatically quotes parameters (depending on the type of the parameter).
+If you don't want that use `unquotedParameter(placeholder, value, description)` instead.
+ 
+### Mitigations
+
```

Comment:
Same here, maybe we can add a small explanation. Maybe something like:

"The mitigations describe those actions the user can follow to overcame the error."

(fell free to take the suggestion or not, and to re-word whatever you think better)